Userland/DevTools/Profiler/Process.h

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
Moving this stuff to separate files would have been nice as a separate commit instead of mixing it in with the rest of this already large change. :)

Reply:
Yeah, sorry about that. By the time I realized it was already too late and I didn't feel like picking apart the patch. :P